plugins/net_plugin/net_plugin.cpp

```diff
@@ -2229,12 +2229,13 @@ namespace eosio {
 
    void dispatch_manager::expire_txns() {
       size_t start_size = 0, end_size = 0;
+      fc::time_point_sec now{time_point::now()};
 
       std::unique_lock<std::mutex> g( local_txns_mtx );
       start_size = local_txns.size();
       auto& old = local_txns.get<by_expiry>();
       auto ex_lo = old.lower_bound( fc::time_point_sec( 0 ) );
```

Comment:
Why was `fc::time_point_sec( 0 )` not moved outside of lock like `fc::time_point_sec now{time_point::now()};`?

Reply:
The compiler will likely optimize that away. The `now()` call I thought time consuming enough to move outside.